src/MDMTransitionContext.h

```diff
@@ -80,3 +80,9 @@ NS_SWIFT_NAME(TransitionContext)
 @property(nonatomic, strong, readonly, nonnull) UIView *containerView;
 
 @end
+
+NS_SWIFT_NAME(InteractiveTransitionContext)
+@protocol MDMInteractiveTransitionContext
```

Comment:
Checking in here again: were you able to try making this conform to `MDMTransitionContext`?